funnel/assets/js/schedule_view.js

```diff
@@ -207,7 +207,10 @@ const Schedule = {
           } else {
             // Scroll page to schedule table
             Utils.animateScrollTo(
-              $(schedule.config.parentContainer).offset().top
+              $(schedule.config.parentContainer)
+                .find('.schedule__date')
+                .last()
+                .offset().top - this.headerHeight
```

Comment:
This should jump to _first_ future date, not last date. Some projects may have future dates added in advance.